sharding-jdbc-example/broadcast-table-example/pom.xml

```diff
@@ -25,6 +25,7 @@
         <module>broadcast-table-raw-jdbc-example</module>
         <module>broadcast-table-spring-boot-example</module>
         <module>broadcast-table-spring-namespace-example</module>
+        <module>broadcast-table-raw-jdbc-multi-example</module>
```

Comment:
`broadcast-table-raw-jdbc-multi-example` can be removed now.

Reply:
ok. Sorry for that . I thought it could be deleted automatically by IDEA... I have remove that.